### PR TITLE
Create check_manually.sh

### LINKED ISSUE #115: What about automating testing normal links, too?
I've made a short script:
```
#!/bin/bash

LINKS=$(grep "http" "$SOURCE_FILE" \
| grep -oP "http.*" \
| sed "s|)$||" \
| sed "s|) .*||" \
| grep -v img.shields.io \
| grep -v travis-ci.org)

mapfile -t LINKS <<< "$LINKS"

for link in "${LINKS[@]}"
do
    echo "Testing $link"
    STATUS_CODE="$(curl -LI "$link" -o /dev/null -w '%{http_code}\n' -s)"
    if [[ "$STATUS_CODE" != "200" ]]
    then
        FALSE_LINKS+=("$link")
    fi
done

if [ -n "${FALSE_LINKS[*]}" ]
then
    echo "Some links weren't reachable."
    for link in "${FALSE_LINKS[@]}"
    do
        echo "$link"
    done
    exit 1
fi
```

### PR DESCRIPTION
**Pull request by szaimen**
fix #115

I think automatic checking isn't really possible but maybe also not necessary to run every time.

### CODE REVIEW

**Comment by szaimen** [comment]
btw: running this still shows:
```
Some links weren't reachable.
http://vanillamusic.io/
```

**Comment by offa** [comment]
> I think automatic checking isn't really possible but maybe also not necessary to run every time.

Agreed, running it from time to time should be enough.

File: check_manually.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+wget https://raw.githubusercontent.com/offa/android-foss/master/README.md
```
Comment (offa) [inline check_manually.sh:3]: This shouldn't be necessary when called locally any more, but I can fix this afterwards.

**Comment by offa** [comment]
Thanks for this contribution :+1:

**Pull request merged**